tools/clang/test/CodeGenSPIRV/spirv.debug.commit.hlsl

```diff
@@ -0,0 +1,6 @@
+// Run: %dxc -T vs_6_0 -E main -fspv-target-env=vulkan1.1 -Zi
+
+// CHECK: OpModuleProcessed "dxc-commit-hash:
```

Comment:
Maybe you should add a check that it does in fact print something after the colon ?

Reply:
Good catch! Fixed.